tests/test_inclined_exponential.py

```diff
@@ -29,51 +33,155 @@
     path, filename = os.path.split(__file__)
     sys.path.append(os.path.abspath(os.path.join(path, "..")))
     import galsim
+    
+save_profiles = True
 
 # set up any necessary info for tests
 # Note that changes here should match changes to test image files
 image_dir = './inclined_exponential_images'
 
 # Values here are strings, so the filenames will be sure to work (without truncating zeros)
+
+# InclinedExponential test cases, which we'll also test InclinedSersic with
 fluxes = ("1.0", "10.0", "0.1", "1.0", "1.0", "1.0")
 image_inc_angles = ("0.0", "1.3", "0.2", "0.01", "0.1", "0.78")
 image_scale_radii = ("3.0", "3.0", "3.0", "3.0", "2.0", "2.0")
 image_scale_heights = ("0.3", "0.5", "0.5", "0.5", "1.0", "0.5")
 image_pos_angles = ("0.0", "0.0", "0.0", "0.0", "-0.2", "-0.2")
+
+# InclinedSersic-only test cases
+sersic_image_ns = ("1.0","1.5","2.0","1.5","2.5","2.5","1.5","1.5","2.5","2.5")
+sersic_image_inc_angles = ("0.0","0.0","0.0","0.0","0.0","0.0","0.1","0.1","0.1","0.1")
+sersic_image_scale_radii = ("3.0","3.0","3.0","3.0","3.0","3.0","2.0","2.0","2.0","2.0")
+sersic_image_scale_heights = ("1.0","1.0","1.0","1.0","1.0","1.0","1.0","1.0","1.0","1.0")
+sersic_image_trunc_factors = ("20","20","20","4.5","20","4.5","20","4.5","20","4.5")
+sersic_image_pos_angles = ("0.0","0.0","0.0","0.0","0.0","0.0","-0.2","-0.2","-0.2","-0.2")
```

Comment:
These values don't have a huge variety in values.  Like all those with `inc_angle` != 0 have the same `inc_angle` and `pos_angle`.  Also, there are no `trunc=0` cases and all the scale heights are equal (and 1 even, which is generally a number to be avoided, since it's easy to miss bugs when 1's are involved).  And finally, I think you use flux=1 for these, which also makes it easy to miss bugs related to the flux normalization.